public/js/index.js

```diff
@@ -1572,7 +1572,27 @@ function initEditor() {
         });
     }).trigger('keyup');
 
-    $('#commit-button').click(function (event) {
+    // Using events from https://github.com/codedance/jquery.AreYouSure#advanced-usage
+    // to enable or disable the commit button
+    const $commitButton = $('#commit-button');
+    const $editForm = $('.ui.edit.form');
+    const dirtyFileClass = 'dirty-file';
+
+    // Disabling the button at the start
+    $commitButton.prop('disabled', true);
+
+    // Registering a custom listener for the file path and the file content
+    $editForm.areYouSure({
+        slient: true,
```

Comment:
```suggestion
        silent: true,
```

Reply:
You're correct. I'm very sorry that I missed that mistake too.  ):
Sadly I didn't notice it, because there's already a quit confirm dialogue by the default `.areYouSure` and I just read over it.
But the fix will be soon up.

Reply:
No problem at all, thanks for taking the time to contribute. 😃